### ISSUE TITLE
Import a3042e361e: Reverted flight on Vive but needs testing (by Caity)

### ISSUE THREAD

**Issue by daleglass** [overte-org/tivolicloud#156]
This is an automated proposal to look at a commit made by Caity and import it into [Overte](https://github.com/overte-org/overte)

| **Commit:** |  a3042e361eabe2d15404f980155a5a06d4073d1a |
| --- | --- |
| **Author:** | Caity |
| **Date:** | Thu, 06 Feb 2020 16:33 |

```
Reverted flight on Vive but needs testing


```

**Stats:**

| **Filename** | Stats | Lines | Added | Removed | Lines in blame |
| --- |  --- | --- | --- | --- | --- |
| interface/resources/controllers/vive.json |  | 10 | 10 | 0 | ⚠ 0
| 1 files | - | 10 | 10 | 0 | 0 |

To work on this, please assign the issue to yourself, then look at the commit and decide whether this would be a good addition to Overte.

* If the commit is useful, tag it with "Tivoli: Keep", and keep it open until it's merged.
* If the commit is not useful, tag it with "Tivoli: Discard", and close it.
* If the commit is not useful right now, but might be later, tag it with "Tivoli: Maybe later", and close it.
* If it's hard to decide, tag it with "Tivoli: Discuss", and keep it open.

Useful commits should be submitted as a [PR against Overte](https://github.com/overte-org/overte/pulls). Tag this issue in PR, so that it's automatically closed once the PR is merged.

You can cherry-pick this issue with this command:

    git cherry-pick a3042e361eabe2d15404f980155a5a06d4073d1a

**Comment by daleglass** [comment]
Duplicate of #136
